### ISSUE TITLE
IOS Build failure with this plugin

### ISSUE THREAD

**Issue by pmwisdom** [EddyVerbruggen/cordova-plugin-googleplus#168]
Getting these errors when building for IOS.

````
Undefined symbols for architecture i386:     -
  "_SecItemAdd", referenced from:
      -[GTMOAuth2Keychain setPassword:forService:accessibility:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemAdd)
  "_SecItemCopyMatching", referenced from:
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemCopyMatching)
  "_SecItemDelete", referenced from:
      -[GTMOAuth2Keychain removePasswordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l091 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemDelete)
  "_SecRandomCopyBytes", referenced from:
      l064 in GoogleSignIn(GIDSignIn.o)
     (maybe you meant: _fbsdkdfl_SecRandomCopyBytes)
  "_SecTrustEvaluate", referenced from:
      l154 in GoogleSignIn(GTMSessionFetcher.o)
  "_kSecAttrAccessible", referenced from:
      -[GTMOAuth2Keychain setPassword:forService:accessibility:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      -[GTMOAuth2Keychain setPassword:forService:accessibility:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly", referenced from:
      +[GTMOAuth2ViewControllerTouch saveParamsToKeychainForName:accessibility:authentication:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l017 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2ViewControllerTouch saveParamsToKeychainForName:accessibility:authentication:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l017 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrAccount", referenced from:
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrGeneric", referenced from:
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrService", referenced from:
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecClass", referenced from:
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecClassGenericPassword", referenced from:
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      +[GTMOAuth2Keychain keychainQueryForService:account:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecMatchLimit", referenced from:
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecMatchLimitOne", referenced from:
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecRandomDefault", referenced from:
      l064 in GoogleSignIn(GIDSignIn.o)
  "_kSecReturnData", referenced from:
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      -[GTMOAuth2Keychain passwordForService:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecValueData", referenced from:
      -[GTMOAuth2Keychain setPassword:forService:accessibility:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
      -[GTMOAuth2Keychain setPassword:forService:accessibility:account:error:] in GoogleOpenSource(GTMOAuth2ViewControllerTouch.o)
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
````

Tried removing the Google Open Source Framework and get these.

````
Undefined symbols for architecture i386:     -
  "_SecItemAdd", referenced from:
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemAdd)
  "_SecItemCopyMatching", referenced from:
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemCopyMatching)
  "_SecItemDelete", referenced from:
      l091 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
     (maybe you meant: _fbsdkdfl_SecItemDelete)
  "_SecRandomCopyBytes", referenced from:
      l064 in GoogleSignIn(GIDSignIn.o)
     (maybe you meant: _fbsdkdfl_SecRandomCopyBytes)
  "_SecTrustEvaluate", referenced from:
      l154 in GoogleSignIn(GTMSessionFetcher.o)
  "_kSecAttrAccessible", referenced from:
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrAccessibleAfterFirstUnlockThisDeviceOnly", referenced from:
      l017 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrAccount", referenced from:
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrGeneric", referenced from:
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecAttrService", referenced from:
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecClass", referenced from:
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecClassGenericPassword", referenced from:
      l088 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecMatchLimit", referenced from:
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecMatchLimitOne", referenced from:
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecRandomDefault", referenced from:
      l064 in GoogleSignIn(GIDSignIn.o)
  "_kSecReturnData", referenced from:
      l090 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
  "_kSecValueData", referenced from:
      l092 in GoogleSignIn(GTMOAuth2ViewControllerTouch.o)
ld: symbol(s) not found for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)

````

**Comment by pmwisdom** [comment]
I downgraded to 4.0.0 and the build completes successfully

**Comment by benconnito** [comment]
I have run into the same problem. The issue is this plugin requires the Security.framework and it was commented out in 2ded53244cd4400e0893d58f109d8edc17241503

**Comment by EddyVerbruggen** [comment]
Thanks for finding this issue and a solution! Fixed in 4.0.7.
